wdl/src/main/scala/wdl/examples/ex7.scala

```diff
@@ -37,7 +38,7 @@ object ex7 {
     ns.taskCalls.find( _.unqualifiedName == "a") foreach { call =>
       val wdlFunctions: CustomFunctions = new CustomFunctions
       val evaluatedInputs = call.evaluateTaskInputs(inputs, wdlFunctions).get
-      println(call.task.instantiateCommand(evaluatedInputs, wdlFunctions).get)
+      println(call.task.instantiateCommand(evaluatedInputs, wdlFunctions).toTry.get)
```

Comment:
I see that this is in `src/main`, but it feels like more of a test to me? Can we move it?

Reply:
That seems out of scope for what this PR is trying to do.

Reply:
There's also the larger concern that all those examples are demonstrating usage of an API we might not want to people to use anymore.